service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceMethodClientOptions.java

```diff
@@ -0,0 +1,39 @@
+package com.microsoft.azure.sdk.iot.service.devicetwin;
+
+import com.microsoft.azure.sdk.iot.service.ProxyOptions;
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * Configurable options for all device method operations.
+ */
+@Builder
+public class DeviceMethodClientOptions
```

Comment:
Is this a customer request?

Reply:
The builder pattern is just for consistency with our RegistryManagerOptions class. These options are a customer ask to configure, yes